src/impl/project/diff/diffutils.ts

```diff
@@ -129,28 +184,225 @@ export default class DiffUtil {
         finalPath = finalPath.trim();
         finalPath = finalPath.replace("\\303\\251", "é");
 
+        let revisionPart = lineParts[0].split(/\t|\s/);
+
         if (deletedFileRegEx.test(fileContents[i])) {
           //Deleted
-          diffFile.deleted.push(finalPath);
+          diffFile.deleted.push({
+            revisionFrom: revisionPart[2].substring(0, 9),
+            revisionTo: revisionPart[3].substring(0, 9),
+            path: finalPath,
+            status: "D"
+          });
         } else {
           // Added or edited
-          diffFile.addedEdited.push(finalPath);
+          let status = "A";
```

Comment:
Magic constants.. Lets have a string constant somewhere denoting what it is

Reply:
I removed those constants